src/main/java/sirius/biz/translations/jdbc/SQLTranslations.java

```diff
@@ -29,6 +28,15 @@ public SQLTranslations(BaseEntity<?> owner) {
         super(owner);
     }
 
+    @Override
+    protected void deleteAllTranslations() {
+        for (SQLTranslation t : oma.select(SQLTranslation.class)
```

Comment:
It's possible to delete all in a single command based on filter instead of looping and deleting one by one.